kinesis-video-pic/src/utils/src/Mutex.c

```diff
@@ -294,9 +282,15 @@ STATUS defaultConditionVariableWait(CVAR cvar, MUTEX mutex, UINT64 timeout)
         CHK(0 == (retVal = pthread_cond_wait(cvar, pMutex)), STATUS_WAIT_FAILED);
     }
     else {
+#ifdef __MACH__
```

Comment:
Is there an advantage using non-portable API for mac?

Reply:
I am using a MBP with macosx, sure want this to be supported also.

Reply:
MBP? = Mack Book Pro? I believe the existing code should support it - I also use macbook pro as one of my development devices

Reply:
yep MBP = MacBookPro, posix pthread does not defined/support monotonic time.
gnu gcc added the extension, and apple added its own extension.
therefore we call different api call based on build environment.